plugins/source/aws/resources/services/s3/bucket_cors_rules.go

```diff
@@ -13,7 +13,6 @@ func BucketCorsRules() *schema.Table {
 		Description: `https://docs.aws.amazon.com/AmazonS3/latest/API/API_CORSRule.html`,
 		Resolver:    fetchS3BucketCorsRules,
 		Transform:   transformers.TransformWithStruct(&types.CORSRule{}),
-		Multiplex:   client.AccountMultiplex,
```

Comment:
What's the reason for removing the `AccountMultiplex`es? Is it related to the the bug fix?

Reply:
I think it's from a previous hypothesis that this was causing duplicates, but in reality it doesn't matter: `Multiplex` is completely ignored for child tables